vega-lite-schema.json

```diff
@@ -1025,6 +1025,12 @@
             },
             "type": "object"
         },
+        "Dict<SelectionSpec>": {
```

Comment:
I think if we give this type and alias, we should be fine. The generated schema is not incorrect so I don't think there is anything to fix in the schema generator. I can take care of this.